### DISCUSSION TITLE
Nulleinspeisung trotz vollem Akku mit Solar-Passthrough / oberem Ladelimit?

### DISCUSSION THREAD

**Discussion by NikGro** [hoylabs/OpenDTU-OnBattery#761] [Q&A] [answered]
Huhu! Eine kurze Frage:

Gibt es eine Möglichkeit die Einstellungen so vorzunehmen, dass ich ein oberes Ladelimit einstellen kann und trotzdem nicht ins Netz einspeise?

Aktuell ist meine Beobachtung, dass ich, wenn ich ein festes SOC-Limit mit 90% einstellen will (zum Akku schonen), ich dazu Solar-Passthrough aktivieren muss. So weit - so gut. 

Nun würde ich ein Einspeisen jedoch gerne um jeden Preis vermeiden, einfach weil ich es dem gierigen Netzbetreiber nicht gönne.  Mit Solar-Passthrough wird die Nulleinspeisung nun aber ignoriert wenn der Akku voll ist und es geht aller Überschuss ins netz.

Gibt es eine "Best of both worlds"-Option, die sowohl das obere SOC-Limit als auch die weitere Nulleinspeisung ermöglicht? Wäre so eine Funktion villeicht für die Zukunft denkbar?

**Comment by spcqike** [comment]
Gibt’s. Aber nur im laderegler. Und der kann von der openDTU aktuell nicht gesteuert werden. 

Du musst beim laderegler eine obere Spannung vorgeben. Dann hört er auf den Akku zu laden. 

Die DTU regelt die Ausgangsleistung des wechselrichters. Der verbraucht Strom. Die Akku Spannung fällt und der laderegler lädt wieder. 

Ob das in Summe schonender ist … keine Ahnung. 

Du kannst aber in openDTU nicht einfach sagen „lade auf 90% und mache dann nur noch eine Nulleinspeisung“. Irgendwo muss die Leistung, die der laderegler an die Batterie weiter reichen will, ja hin. Im Solar passthrough „verschwindet“ die Leistung einfach in der AC Einspeisung. Daher kann der Akku auf dem gewünschten SoC gehalten werden. 

Ohne dynamische Begrenzung der Ausgangsleistung des ladereglers ist deine Frage also mit „Nein“ zu beantworten.

**Reply by NikGro** [reply]
Das war auch meine erste Vermutung, allerdings klappt das ganze ohne passthrough super, sprich: wenn der Akku voll ist und keinen strom mehr annimmt, wird dennoch die Nulleinspeisung gehalten.

Vielleicht habe ich auch einen Denkfehler! Ich hatte nur anhand meiner beobachtung den Eindruck, dass die DTU den MPPT zwar regeln kann, es aber an der Umsetzung ausserhalb vom passthrough fehlt.

Allerdings bin ich nicht besonders tief in der Materie, vielleicht hatte das ja auch andere Gründe!

**Comment by schlimmchen** [comment, marked as answer]
Naja, dann stell deinen Laderegler so ein, dass er in Absorbtion und später in Float geht, wenn dein Akku bei ca. 90% ist. Das geht natürlich nur über die Spannung, die musst du dann genau tunen.

Den Full-Solar-Passthrough SoC setzte du dann auf 100%, die du nie erreichst.

**Reply by NikGro** [reply]
Die Möglichkeit war mir nicht bewusst bzw. ja vielleicht genau die Info die mir gefehlt hat! Von den Spannungen habe ich bisher mangels Wissens die Finger gelassen!

Hättest du eine Empfehlung für eine "Startwert"? Habe eine Pylontech US2000C verbaut. Das Herantasten würde ich in jedem Fall versuchen wollen!

**Reply by NikGro** [reply]
Für mich als Laien wäre es natürlich eine Prima Option das ganze auch einfach via SOC einstellen zu können, falls sowas für zukünftige Versionen denkbar bzw überhaupt umsetzbar wäre

**Reply by schlimmchen** [reply]
Du hast mich offensichtlich nicht richtig verstanden... Du sollst **im Laderegler** einstellen, bis zu welcher Spannung er mit Vollgas auflädt, sodass den Akku höchsten 90% voll wird.

**Reply by NikGro** [reply]
Ah! Ja ich sehe das Missverständnis! Ich lese mir mal durch wie das beim MPPT funktioniert!

Wie funktioniert das wenn der Akku voll ist? Erkennt der MPPT dann den Widerstand und regelt selbstständig herunter?

**Reply by spcqike** [reply]
Stichwort CC-CV Ladung. 

Der laderegler stellt am Ende eine feste Spannung ein. Die, die du ihm vorgibst. Nähert sich die batteriespannung dieser, kann sie weniger Energie aufnehmen. Der Ladestrom fällt. Bis er 0 wird weil beide Spannungen gleich sind.

**Reply by NikGro** [reply]
Okay, ich kann folgen! Ich hoffe meine Verständnisfragen stören nicht.

Nun sind Inverter und Batterie ja beide hinter dem MPPT zusammen angeschlossen, also beide jeweils für +/- an einer Kabelbrücke. Ändert das etwas an der Art und Weise wie der MPPT diese Spannung von der Batterie erkennt?

**Reply by spcqike** [reply]
Nö. Zieht der inverter Strom, bricht die Batterie ein. Der MPPT schiebt nach. Das ganze passiert aktuell auch bei pass through. Daher ist eine Batterie als Puffer, wenn es auch nur eine kleine ist, zwingend notwendig. Eine gewisse Ladung und Entladung hast du halt immer. Um ein gewisses Level.

**Reply by NikGro** [reply]
Okay!

Also vorgehen für mein Ziel:

MPPT via USB an Laptop und in der Konfig die Float-Voltage tunen bis der MPPT von sich aus verweigert den Akku über 90% zu laden. DTU hat dann quasi nichts mehr mit meinem Vorhaben zu tun.

Korrekt?

Wäre es in Zukunft möglich bzw zumindest denkbar den MPPT via VEDirect von der DTU in Float setzen zu lassen? Oder die Float-Voltage aus dem UI heraus anzupassen?
